src/components/ReactTags.js

```diff
@@ -315,8 +315,7 @@ class ReactTags extends Component {
       );
 
       if (
-        (this.props.autocomplete === 1 && possibleMatches.length === 1) ||
-        (this.props.autocomplete === true && possibleMatches.length)
+        this.props.autocomplete && possibleMatches.length === 1
```

Comment:
@ZebGirouard can you add spec for this as well ?

Reply:
@ZebGirouard any update ?

Reply:
Hi @ad1992 , I changed line 318 as requested, am I missing something?

Reply:
Oh, are you saying I need to add a test?  Not sure how to do that as I'm new to this project....